### PR TITLE
feat(console): add callback uri field

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Add "Callback URI" field, there is a special case for creating non-standard connector: the connector id is pre-generated in front-end.

## Checklist

- [ ] `.changset-staged`
- [ ] unit tests
- [ ] integration tests
- [ ] docs

OR

- [x] This PR is not applicable for the checklist

截屏2023-02-27 上午11 09 01

截屏2023-02-27 上午11 04 44

截屏2023-02-27 上午11 09 14

### CODE REVIEW

File: packages/console/src/pages/Connectors/components/Guide/index.tsx [hunk 1]
```diff
@@ -58,6 +61,14 @@ const Guide = ({ connector, onClose }: Props) => {
     watch,
   } = methods;
 
+  useEffect(() => {
+    const { isStandard } = connector;
+
+    if (isStandard) {
+      setCallbackConnectorId(generateStandardId());
+    }
```
Comment (darcyYe) [inline packages/console/src/pages/Connectors/components/Guide/index.tsx:69]: Should we check `platform` and `setCallbackConnectorId` when it's a social connector?

File: packages/console/src/pages/Connectors/components/Guide/index.tsx [hunk 2]
```diff
@@ -70,6 +81,7 @@ const Guide = ({ connector, onClose }: Props) => {
     const basePayload = {
       config,
       connectorId,
+      id: conditional(isStandard && callbackConnectorId),
```
Comment (darcyYe) [inline packages/console/src/pages/Connectors/components/Guide/index.tsx:84]: Should check whether the connector is a social connector?

File: packages/console/src/pages/Connectors/components/Guide/index.tsx (hunk 1 above)
Comment (xiaoyijun) [reply packages/console/src/pages/Connectors/components/Guide/index.tsx:69]: Agree.

File: packages/console/src/pages/Connectors/components/Guide/index.tsx (hunk 2 above)
Comment (wangsijie) [reply packages/console/src/pages/Connectors/components/Guide/index.tsx:84]: updated

File: packages/console/src/pages/Connectors/components/Guide/index.tsx (hunk 1 above)
Comment (wangsijie) [reply packages/console/src/pages/Connectors/components/Guide/index.tsx]: updated

**Review by xiaoyijun — APPROVED** [review]

**Review by darcyYe — APPROVED** [review]

**Pull request merged**